content/en/continuous_integration/setup_pipelines/buildkite.md

```diff
@@ -33,6 +33,33 @@ The [Pipelines][3] and [Pipeline Executions][4] pages populate with data after t
 
 **Note**: The Pipelines page shows data for only the default branch of each repository.
 
+## Setting custom tags
+
+Custom tags can be added to buildkite traces by using the `buildkite-agent meta-data set` command.
+Any meta-data tags with a key starting by `dd_tags.` are added to the job and pipeline spans.
+
+The YAML below illustrates a simple pipeline where the following custom tags are added to the
+resulting trace in Datadog:
+
+- `team: backend`
+- `go.version: go version go1.17 darwin/amd64` (output will depend on the runner)
+
+The tags are available in the root span of the trace as well as the specific job in
+which the tag was created.
+
+```yaml
+steps:
+  - command: buildkite-agent meta-data set "dd_tags.team" "backend"
+  - command: go version | buildkite-agent meta-data set "dd_tags.go.version"
+    label: Go version
+  - commands: go test ./...
+    label: Run tests
+```
```

Comment:
Can we switch this around? So we first put the pipeline definition and then what the result is on CI app